### ISSUE TITLE
RKE2 provisioning: Fail whale directly after cluster provisioning is finished

### ISSUE THREAD

**Issue by bashofmann** [rancher/rancher#34259]
If you navigate to the cluster explorer of a new cluster directly after it finished provisioning and marked active in the UI, the cluster explorer does not work and a fail whale error page is displayed instead.

The cluster explorer only starts working after about 30 to 60s.

Tested with Rancher v2.6-rc2

**Comment by deniseschannon** [comment]
@bashofmann If you can try to reproduce with the latest v2.6-head, that'd be great!

**Comment by bashofmann** [comment]
On a first quick test, it's better, but not perfect. The cluster is connected and accessible for a bit, and then after about 30 seconds becomes disconnected again for another 30 seconds. Now the cluster is grayed out during this time though. Once the cluster becomes available in the UI again after that, it's reliably accessible.

**Comment by davidnuzik** [comment]
**My checks PASSED**
 
**Reproduction Steps:**

Not required. I reviewed Bastian's videos and observed and understand the issue. I've also seen it before personally.
 
**Validation Steps:**

Rancher version: v2.6-head `72986b2e4` pulled at 8/17/21 1:32 pm Pacific
Rancher cluster type: single-node docker
Docker version: 20.10
 
Downstream cluster type: rke2
Downstream K8s version: v1.21.3-rc6+rke2r2
 
Validation steps:

1. Create any RKE2 cluster. Stay on the cluster management page. Watch the left side pane for the cluster to be accessible for the cluster explorer.
Note: It is still accessible just before the cluster becomes active. But the cluster seems to quickly become "Active" right after I can click on the cluster in cluster explorer.
2. Click on the cluster in the cluster explorer before the cluster is active. No issues observed (no fail wale).
 
Additional Info:
I tried to cause the issue twice and couldn't cause it either time.
1st time a 1-node cluster, 2nd time a multi-node cluster with several machine pools (shouldn't matter but figure I'd try). Worked fine in both cases.
